src/test/java/domain/PurchasedLottosTest.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import domain.strategy.CustomPurchaseStrategy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@DisplayName("상금과 수익률을 옳바르게 계산하는지 테스트한다.")
```

Comment:
```suggestion
@DisplayName("상금과 수익률을 올바르게 계산하는지 테스트한다.")
```

다만, 여기에까지 이름이 있을 필요가 있는지 의문이긴 하네요. 각 테스트 메서드에 이름을 잘 붙여주는 것이 더 좋지 않을까요?

Reply:
일단 오타는 수정했습니다!!

테스트 클래스 별로 붙이는 `@DisplayName`의 경우 초기에는 붙이지 않았으나 클래스별로 각각 어떤 테스트 내용들이 있는지 알리고자 붙여봤습니다.

각각의 파일명에서 `@DisplayName`을 붙여두니 어떤 테스트들이 존재하는지 확실히 알 수 있어서 더 좋다고 생각하였습니다. 하지만 더 깊게 생각해보니 현재는 미션의 규모가 작아 테스트 오류가 발생하여도 어떤 파일에서 오류가 발생하였는지 쉽게 찾을 수 있는데 프로젝트 규모가 커져서 테스트 파일만 100개이상 존재하게 된다면 재정의한 `@DisplayName`으로 어떤 파일에서 테스트 실패가 나타나는지 확인하기 힘들거같다는 생각이 드네요😅

실무에서는 보통 테스트 클래스 위에는 DisplayName을 붙이지 않고 테스트 메서드 위에만 붙이는 편인가요?? 
InputValidationTest.java파일에서 사용한 Nested Test의 경우도 붙이지 않는지 궁금합니다🧐